targets/esp-idf/module.c

```diff
@@ -21,6 +21,7 @@
 #include <sys/stat.h>
 #include <unistd.h>
 
+#include "jerryscript-core.h"
```

Comment:
jerryscript-core.h is internal, jerryscript.h should be used instead. Please also fix the commit message.